### PR TITLE
JCRVLT-521 drop support of Jackrabbit 2.14, Oak 1.6 and Declarative

### PR DESCRIPTION
**Pull request by kwin**
Services 1.3

### CODE REVIEW

**Review by tripodsan — COMMENTED** [review]
I cannot judge the PR. @DominikSuess can you review ?

File: parent/pom.xml [hunk 1]
```diff
@@ -517,12 +517,12 @@ Bundle-Category: jackrabbit
             <dependency>
                 <groupId>org.apache.commons</groupId>
                 <artifactId>commons-lang3</artifactId>
-                <version>3.9</version>
+                <version>3.6</version>
```
Comment (reschke) [inline parent/pom.xml:520]: why the downgrade?

File: parent/pom.xml [hunk 2]
```diff
@@ -532,12 +532,12 @@ Bundle-Category: jackrabbit
             <dependency>
                 <groupId>org.apache.httpcomponents</groupId>
                 <artifactId>httpmime</artifactId>
-                <version>4.5.3</version>
+                <version>4.5.8</version>
```
Comment (reschke) [inline parent/pom.xml:535]: These should use the most recent version (4.5.13)

File: parent/pom.xml (hunk 1 above)
Comment (kwin) [reply parent/pom.xml:520]: In order to be still compatible with AEM 6.4. For OSGi bundles you rather reference the minimum version you are compatible with.

File: parent/pom.xml (hunk 2 above)
Comment (kwin) [reply parent/pom.xml:535]: Same as above

**Pull request merged**